### PR TITLE
14. FileReader.readAsArrayBuffer() [ES]

### LINKED ISSUE #4021: 14. FileReader.readAsArrayBuffer() [ES]

### PR DESCRIPTION
**Pull request by JuanVqz**
- [x] change from HTML to MD
- [x] update the whole content file

Closes #4021

### CODE REVIEW

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Sintaxis
+
+```js
+instanceOfFileReader.readAsArrayBuffer(blob);
+```
+
+### Parámetros
+
+- `blob`
+  - : The {{domxref("Blob")}} or {{domxref("File")}} from which to read.
```
Comment (JuanVqz) [inline files/es/web/api/filereader/readasarraybuffer/index.md:35]:
```suggestion
  - : El {{domxref("Blob")}} o {{domxref("File")}} desde el que se leerá.
```

**Review by Vallejoanderson — APPROVED** [review]

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - Reference
+  - readAsArrayBuffer
+browser-compat: api.FileReader.readAsArrayBuffer
+translation_of: Web/API/FileReader/readAsArrayBuffer
+---
+{{APIRef("File API")}}
+
+El método **`readAsArrayBuffer()`** de la interfaz {{domxref("FileReader")}}
+se utiliza para comenzar a leer el contenido especificado en {{domxref("Blob")}}
+o {{domxref("File")}}. Cuando la operación de lectura finaliza,
```
Comment (davbrito) [inline files/es/web/api/filereader/readasarraybuffer/index.md:20]:
```suggestion
se utiliza para comenzar a leer el contenido de un {{domxref("Blob")}}
o {{domxref("File")}} especificado. Cuando la operación de lectura finaliza,
```

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+el método {{domxref("FileReader.readyState","readyState")}} se convierte en `DONE`,
```
Comment (davbrito) [inline files/es/web/api/filereader/readasarraybuffer/index.md:21]: No es un método. Es una propiedad.

```suggestion
la propiedad {{domxref("FileReader.readyState","readyState")}} se convierte en `DONE`,
```

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+y el evento {{domxref("FileReader/loadend_event", "loadend")}} es lanzado. Al mismo tiempo,
```
Comment (davbrito) [inline files/es/web/api/filereader/readasarraybuffer/index.md:22]:
```suggestion
y se desencadena el evento {{domxref("FileReader/loadend_event", "loadend")}}. En ese momento,
```

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+### Parámetros
+
+- `blob`
+  - : El {{domxref("Blob")}} o {{domxref("File")}} desde el que se leerá.
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad del navegador
```
Comment (davbrito) [inline files/es/web/api/filereader/readasarraybuffer/index.md:41]:
```suggestion
## Compatibilidad con navegadores
```

File: files/es/web/api/filereader/readasarraybuffer/index.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+{{Compat}}
+
+## Ver también
```
Comment (davbrito) [inline files/es/web/api/filereader/readasarraybuffer/index.md:45]:
```suggestion
## Véase también
```

**Review by davbrito — CHANGES_REQUESTED** [review]

File: files/es/web/api/filereader/readasarraybuffer/index.md (hunk 3 above)
Comment (JuanVqz) [reply files/es/web/api/filereader/readasarraybuffer/index.md:21]: The [FileReader](https://developer.mozilla.org/en-US/docs/Web/API/FileReader) interface's readAsArrayBuffer() method is used to start reading the contents of a specified [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) or [File](https://developer.mozilla.org/en-US/docs/Web/API/File).

@davbrito según yo si es metodo, donde dice propiedad?

https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsArrayBuffer

**Review by Vallejoanderson — APPROVED** [review]

**Review by Vallejoanderson — APPROVED** [review]

**Review by davbrito — APPROVED** [review]

**Pull request merged**